store/config.go

```diff
@@ -59,36 +73,93 @@ func ParseConfig(filePath string, logger hclog.Logger) (*LogzioConfig, error) {
 	var logzioConfig *LogzioConfig
 	if filePath != "" {
 		logzioConfig = &LogzioConfig{}
+		// Set default values
+		logzioConfig.LogCountLimit = defaultLogCountLimit
+		logzioConfig.Compress = true
```

Comment:
for logzio go ( which is logs mainly ) we need the gzip
Did you check if tracing is working with gzip as well
if yes cool 
If not please update to false